rust/connlib/tunnel/src/client.rs

```diff
@@ -490,8 +489,6 @@ impl ClientState {
             .inspect_err(|e| tracing::debug!("Failed to encapsulate: {e}"))
             .ok()??;
 
-        tracing::trace!("Encapsulated packet");
```

Comment:
This ended up being way too noisy and actually useless in real logs so I removed it.

cc @jamilbk :)